modules/rosa-sts-creating-cluster.adoc

```diff
@@ -0,0 +1,194 @@
+  --operator-iam-roles ebs-cloud-credentials,openshift-cluster-csi-drivers,arn:aws:iam::${aws_account_id}:role/ManagedOpenShift-openshift-cluster-csi-drivers-ebs-cloud-credent
+----
++
+.Example output
+[source,terminal]
+----
+I: Creating cluster with identifier '<cluster_id>' and name '<cluster_name>'
+I: To view list of clusters and their status, run `rosa list clusters`
+I: Cluster '<cluster_name>' has been created.
+I: Once the cluster is 'Ready' you will need to add an Identity Provider and define the list of cluster administrators. See `rosa create idp --help` and `rosa create user --help` for more information.
+I: To determine when your cluster is Ready, run `rosa describe cluster <cluster_name>`.
+----
+** To create a cluster with STS using interactive prompts:
++
+[source,terminal]
+----
+$ rosa create cluster --interactive
+----
+
+. Enter the following command to check the status of your cluster and retrieve your cluster ID. During cluster creation, the `State` field from the output will transition from `pending` to `installing`, and finally to `ready`.
++
+[source,terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name>
+----
++
+.Example output
+[source,terminal]
+----
+Name:                       <cluster_name>
+ID:                         <cluster_id>
+External ID:                <external_id>
+OpenShift Version:          <version>
+Channel Group:              stable
+DNS:                        *.openshiftapps.com
+AWS Account:                123456789012
+API URL:                    https://api.<cluster_name>.openshiftapps.com:6443
+Console URL:                https://console-openshift-console.apps.<cluster_name>.openshiftapps.com
+Region:                     <region>
+Multi-AZ:                   false
+Nodes:
+ - Master:                  3
+ - Infra:                   2
+ - Compute:                 2
+Network:
+ - Service CIDR:            172.30.0.0/16
+ - Machine CIDR:            10.0.0.0/16
+ - Pod CIDR:                10.128.0.0/14
+ - Host Prefix:             /23
+STS Role ARN:               arn:aws:iam::123456789012:role/ManagedOpenShift-IAM-Role
+Support Role ARN:           arn:aws:iam::123456789012:role/ManagedOpenShift-Support-Role
+Instance IAM Roles:
+ - Master:                  arn:aws:iam::123456789012:role/ManagedOpenShift-ControlPlane-Role
+ - Worker:                  arn:aws:iam::123456789012:role/ManagedOpenShift-Worker-Role
+Operator IAM Roles:
+ - arn:aws:iam::123456789012:role/ManagedOpenShift-openshift-machine-api-aws-cloud-credentials
+ - arn:aws:iam::123456789012:role/ManagedOpenShift-openshift-cloud-credential-operator-cloud-crede
+ - arn:aws:iam::123456789012:role/ManagedOpenShift-openshift-image-registry-installer-cloud-creden
+ - arn:aws:iam::123456789012:role/ManagedOpenShift-openshift-ingress-operator-cloud-credentials
+ - arn:aws:iam::123456789012:role/ManagedOpenShift-openshift-cluster-csi-drivers-ebs-cloud-credent
```

Comment:
@kmccarron-rh - my bad - this output is not part of the official `v1.0.8` release. I think we should remove it to avoid confusion. What do you think @acathrow, @wgordon17?
```suggestion
```